javascript/javascript3/week1/WeatherApp/main.js

```diff
@@ -0,0 +1,130 @@
+const btn1 = document.getElementById('btnGetCity');
+const btn2 = document.getElementById('btnGetLoc');
+const displayMessage = document.getElementById("displayMessage");
+const displayWeather = document.getElementById("displayWeather");
+
+//onload show weather for saved city or location in localstorage
+window.onload = () => {
+    if (localStorage.getItem("city")) {
+        console.log(localStorage.getItem("city"));
+        const url = `https://api.openweathermap.org/data/2.5/weather?q=${localStorage.getItem("city")}&appid=e75ca172f74f2d4c651c1c428d8fac19&units=metric`
+        fetchWeather(url);
+    } else if (localStorage.getItem("lat") && localStorage.getItem("lon")) {
+        console.log(localStorage.getItem("lat"));
+        console.log(localStorage.getItem("lon"));
```

Comment:
I would either remove these or write something slightly more informative for the user - something like "Loaded latitute and longitude for your location: {lon} {lat}"